Resources/Prototypes/_Sunrise/Entities/Clothing/Eyes/night_vision_device.yml

```
@@ -1,37 +1,77 @@
+- type: entity
+  name: NVD
+  abstract: true
+  id: BaseNightVisionDevice
+  parent: [BaseItem]
+  components:
+  - type: Item
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)